catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/pom.xml

```diff
@@ -163,40 +175,26 @@
             <artifactId>jaxb2-basics-runtime</artifactId>
             <version>0.11.0</version>
```

Comment:
✏️ https://github.com/codice/ddf/blob/master/pom.xml#L226
` <jvnet.jaxb2.version>0.10.0</jvnet.jaxb2.version>`

Reply:
That's a different version.

Reply:
should we stay the same? for SWAP purpose or 0.11.0 is also approved? @bdeining

Reply:
all versions will be bumped to the latest version in a separate PR now that ddf-jaxb 1.0.2 has been released.